pulp_ansible/app/tasks/roles.py

```diff
@@ -50,7 +50,7 @@ def synchronize(remote_pk, repository_pk, mirror=False):
     )
     first_stage = RoleFirstStage(remote)
     d_version = DeclarativeVersion(first_stage, repository, mirror=mirror)
-    d_version.create()
+    return d_version.create()
```

Comment:
I didn't expect this, but I'm ok with it.

Reply:
Same here.  It doesn't change anything since we're not using this result, but the convention is that we *could* be using this result.